nano/nano_node/entry.cpp

```diff
@@ -32,6 +32,9 @@ int main (int argc, char * const * argv)
 		("disable_unchecked_drop", "Disables drop of unchecked table at startup")
 		("fast_bootstrap", "Increase bootstrap speed for high end nodes with higher limits")
 		("batch_size",boost::program_options::value<std::size_t> (), "Increase sideband batch size, default 512")
+		("block_processor_batch_size",boost::program_options::value<std::size_t> (), "Increase block processor transaction batch write size, default 0 (limited by config block_processor_batch_max_time), 256k for fast_bootstrap")
+		("block_processor_full_size",boost::program_options::value<std::size_t> (), "Increase block processor allowed blocks queue size before dropping live network packets and holding bootstrap download, default 65536, 1 million for fast_bootstrap")
+		("block_processor_verification_size",boost::program_options::value<std::size_t> (), "Increase batch signature verification size in block processor, default 0 (2048 * signature checker threads + 1), unlimited for fast_bootstrap")
```

Comment:
Should it say `default 0` here? looks a bit confusing.
Also following BODMAS we should put: `2048 * (signature checker threads + 1)`

Reply:
Is new definition better?
Config is created in `daemon.run ()` after this

Reply:
I misread it actually :P, yeh looks good.

But there is some duplication in the setting of the `node_flags` now in nano_node/nano_wallet entry.cpp now, can this be put into a function accessible to both .exes? (the new `if (flags.fast_bootstrap)` etc..` stuff). Alternatively just create a new function in `nano_nano` for it, as the wallet is being separated out later so these options will be removed anyway I'd imagine. Actually does this even work as we don't set the `description.add_options ()` for nano_wallet anyway?

Reply:
You right, it's not working in wallet *shoked*

Reply:
You right, it's not even working there..

Reply:
Made separate function for node

Reply:
Looks good